release-notes.md

```diff
@@ -169,33 +169,33 @@ Assume a CFO wants to know the cost of all AWS compute instances that are runnin
 
-The query then generates a new metric `instances_hourly_cost_estimate` - a total hourly cost estimate, broken down by account, region and instance type. It’s a simple way to understand which AWS accounts and the teams responsible for the accounts drive compute cost. And the nice thing is that Finance doesn’t have to bother engineering to get these metrics. They can just run the queries themselves in the Cloudkeeper CLI.
+The query then generates a new metric `instances_hourly_cost_estimate` - a total hourly cost estimate, broken down by account, region and instance type. It’s a simple way to understand which AWS accounts and the teams responsible for the accounts drive compute cost. And the nice thing is that Finance doesn’t have to bother engineering to get these metrics. They can just run the queries themselves in the resoto CLI.
 
-Writing queries may not be everyone’s thing though. For those users, we also maintain several pre-configured metrics per resource in Cloudkeeper.
+Writing queries may not be everyone’s thing though. For those users, we also maintain several pre-configured metrics per resource in resoto.
 
-These pre-configured metrics are running as queries in `ckmetrics`. `ckmetrics` connects to the core, runs the queries and recalculates the metrics automatically every time something has changed in the graph, e.g. after a collect or a clean-up. The results are cached in `ckmetrics` and exported to Prometheus where they can be queried via PromQL. From there - you can send them to any visualization tool that understands the prometheus format, such as Grafana. Future versions of `ckmetrics` will allow a user to edit the pre-defined metrics as well as define their own.
+These pre-configured metrics are running as queries in `resotometrics`. `resotometrics` connects to the core, runs the queries and recalculates the metrics automatically every time something has changed in the graph, e.g. after a collect or a clean-up. The results are cached in `resotometrics` and exported to Prometheus where they can be queried via PromQL. From there - you can send them to any visualization tool that understands the prometheus format, such as Grafana. Future versions of `resotometrics` will allow a user to edit the pre-defined metrics as well as define their own.
 
 ##  CLI - From Local To Remote Execution
 
-The old CLI ran locally on a user’s desktop. That implied that two different users would never look at the same version of a graph - because it was their own “local” version that Cloudkeeper had generated at a specific point in time.
+The old CLI ran locally on a user’s desktop. That implied that two different users would never look at the same version of a graph - because it was their own “local” version that resoto had generated at a specific point in time.
```

Comment:
```suggestion
The old CLI ran locally on a user’s desktop. That implied that two different users would never look at the same version of a graph - because it was their own “local” version that Cloudkeeper had generated at a specific point in time.
```